README.md

```diff
@@ -1,5 +1,133 @@
-# objc-xcode-warnings-errors
+# Xcode: Warnings and Errors
+
+<img src="https://curriculum-content.s3.amazonaws.com/ios/ios-objc-fundamentals-unit/xcode_warning_icon.png" width=200>
+<img src="https://curriculum-content.s3.amazonaws.com/ios/ios-objc-fundamentals-unit/xcode_error_icon.png" width=200>
+
+## Objectives
+
+1. Learn the distinction between warning and errors and how to navigate to them with the issue navigator.
+2. Recognize common issues and how to fix them:
+  * Unused variable
+  * Incompatible integer to pointer conversion
+  * The return statement ends the method
+  * Unmatched curly braces `{` `}`
+  * Variables and method calls written outside of a method body
+
+
+## Introduction
+
+When using Xcode, you're going to encounter warnings and errors. This is a normal part of the development process, so don't be alarmed when you see these symbols: they're there to help you!
+
+Warnings and errors are generated by the pre-compiler (though often referred to as just "the compiler"), which is a tool built into Xcode that continuously scans through the code *as it's edited* in order to identify problems.
+
+<img src="https://curriculum-content.s3.amazonaws.com/ios/ios-objc-fundamentals-unit/xcode_warning_icon.png" width=24> A **warning** is a minor issue that something unexpected might be caused by the code, but won't necessarily cause a crash at run time. Warnings will not impede the scheme being built.
+
+<img src="https://curriculum-content.s3.amazonaws.com/ios/ios-objc-fundamentals-unit/xcode_error_icon.png" width=24> An **error** is a major issue that will almost certainly cause a crash at run time, so the error impedes the scheme from being built.
+
+Like all of the tools bundled together in Xcode, the pre-compiler is imperfect. It isn't able to identify every possible problem that might come about, and the messages that accompany these warnings and errors aren't always direct; this often leads Xcode to generate cryptic messages and to suggest incorrect fixes.
+
+Feeling confused is normal, but **don't make the assumption that Xcode knows more than you do.** It's an entire sub-skill within iOS development to be able to interpret the underlying cause of the warning or error message and be able to apply the appropriate change. With experience, you'll come to see Xcode like a canine companion that, while intelligent in its own way, is more capable of "sniffing out" (detecting) problems than it is capable of knowing what do about them like a [mountain guru](http://tvtropes.org/pmwiki/pmwiki.php/Main/HermitGuru). 
```

Comment:
I feel like it's important somehow to stress that warnings aren't to be ignored. While compilation will continue, it is almost always a sign that you've done something weird. Even if Xcode's suggestions on how to fix it are weird/wrong/absent, it's not something to blindly ignore.